Dockerfile

```diff
@@ -23,3 +23,7 @@ WORKDIR /deps
 ADD . /home/jovyan/work
 WORKDIR /home/jovyan/work
 RUN pip install .
+
+# to run tests
+COPY run_tests.sh /home/jovyan/run_tests.sh
```

Comment:
Shouldn't this be covered already by line 23 `ADD . /home/jovyan/work`?

Reply:
You're right!